server/chat-plugins/hangman.ts

```diff
@@ -419,15 +454,25 @@ export const commands: Chat.ChatCommands = {
 			this.checkCan('mute', null, room);
 			if (!hangmanData[room.roomid]) hangmanData[room.roomid] = {};
 			if (!target) return this.parse('/help hangman');
+			const parts = target.split(',').map(p => p.trim()).filter(Boolean);
+			if (parts.length < 1) {
+				return this.errorReply("This is invalid, format using /hangman addrandom word, hint, hint, ...");
```

Comment:
try to use `throw new Chat.ErrorMessage`

Reply:
Also, I'd write it as something like `Correct syntax: /handman addrandom word, hint, hint, ...`